src/client/index.html

```diff
@@ -8,7 +8,7 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,500|Roboto:300,400,500,700"
 		rel="stylesheet" />
-	<link rel="icon" href="./assets/img/favicon.ico" type="image/x-icon" />
+	<link rel="icon" href="https://vandyhacks-logo.s3.us-east-2.amazonaws.com/favicon.ico" type="image/x-icon" />
```

Comment:
why do we need remote url here?

Reply:
wouldn't work with local files here unless introduces new libraries in express or react I guess

Reply:
can't u do something like https://github.com/VandyHacks/vaken/blob/f019663d504177d3f0624302ba7b00a4926b546e/src/client/routes/login/OAuthLogin.tsx#L4